package.json

```diff
@@ -2,6 +2,11 @@
   "scripts": {
     "test": "xo"
   },
+  "dependencies": {
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-webstore-upload": "^0.8.10"
+  },
```

Comment:
OK, so I overcomplicated this, sorry. I moved the grunt-related packages to `devDependencies` and Travis CI will install them without any special instructions. I should have figured this from our `npm test` command running using a `devDependency`. Please check the pushed version.